prisma/schema.prisma

```diff
@@ -7,13 +7,18 @@ generator client {
 
 datasource db {
   provider = "postgresql"
-  url      = env("DATABASE_URL")
+  url      = "postgresql://tirusTimes:1234@localhost:5432/prisma-postgres?schema=public"
```

Comment:
you should use the env not hardcode here

Reply:
I do agree